atomic_reactor/constants.py

```diff
@@ -60,3 +60,14 @@
 )
 
 DEFAULT_DOWNLOAD_BLOCK_SIZE = 10 * 1024 * 1024 # 10Mb
+
+LABEL_ALIASES = {
+    "name" : ('name', 'Name'),
```

Comment:
I mean that code using it ought not to be using a string literal. Eg
```
labels = Labels(df.labels)
name = labels.get_name()
```

or something similar.